app/components/orangelight/search_bar_component.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= form_with url: search_catalog_url, local: true, method: @method, class: @classes.join(' '), scope: @prefix, role: 'search', aria: { label: scoped_t('submit') }, **@form_options do |f| %>
   <%= render Blacklight::HiddenSearchStateComponent.new(params: @params) %>
   <% if search_fields.length > 1 %>
-    <%= f.label :search_field, scoped_t('search_field.label'), class: 'sr-only visually-hidden' %>
+    <%= f.label :search_field, scoped_t('search_field.label'), class: 'visually-hidden visually-hidden' %>
```

Comment:
Looks like the same class is here twice

```suggestion
    <%= f.label :search_field, scoped_t('search_field.label'), class: 'visually-hidden' %>
```

Reply:
oh yes! probably when I replaced sr-only with visually-hidden.